docs/API.md

```diff
@@ -427,12 +427,15 @@ mysql2
 
 ##### Tags
 
-| Tag              | Description                                               |
-|------------------|-----------------------------------------------------------|
-| db.name          | The name of the queried database.                         |
-| db.user          | The user who made the query.                              |
-| out.host         | The host of the PostgreSQL server.                        |
-| out.port         | The port of the PostgreSQL server.                        |
+| Tag              | Description                                                                           |
+|------------------|---------------------------------------------------------------------------------------|
+| component        | The name of the database library/module/package (always set to `postgres`).           |
```

Comment:
Not sure I prefer the boilerplate of `The name of the database library/module/package` to just `always set to "postgres"`.  Otherwise seems like you are at once describing the OT tag and this instrumentation.